llvm/lib/Target/AArch64/AArch64ExpandPseudoInsts.cpp

```diff
@@ -987,7 +987,7 @@ AArch64ExpandPseudo::expandCondSMToggle(MachineBasicBlock &MBB,
   // Expand the pseudo into smstart or smstop instruction. The pseudo has the
   // following operands:
   //
-  //   MSRpstatePseudo <za|sm|both>, <0|1>, pstate.sm, expectedval, <regmask>
+  //   MSRpstatePseudo <za|sm|both>, <0|1>, pstate.sm, condition, <regmask>
```

Comment:
Looking at the changes to the inputs to MSRpstatePseudo in AArch64SMEInstrInfo.td, should `pstate.sm` be removed here?

Reply:
That depends on the condition. If this is a conditional toggle, then the runtime `pstate.sm` value will be passed to the pseudo as a variadic argument. I've updated the comment to make it clear that `pstate.sm` is optional.